### PR TITLE
[chinese] translate embedding section into chinese

### PR DESCRIPTION
**Pull request by chuan9**
Translate Embedding API for android into chinese

### CODE REVIEW

**Comment by chuan9** [comment]
@jondong @lisa0314 Please have a review

File: public/documentation/android/embedding_crosswalk_zh.md [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+
+### 导入Crosswalk Webview到ADT
+
+下一步是通过导入的解压文件Crosswalk Webview bundle,在ADT中创建一个工程。您可以参考下面的工程，编译属于您自己的Crosswalk嵌入模式应用。
+
+创建工程：
+
+1.  打开ADT。
+
+2.  选择*File* > *New* > *Project...*, 然后*Android* > *Android Project From Existing Code*。
```
Comment (jondong) [inline public/documentation/android/embedding_crosswalk_zh.md]: 然后选择

**Comment by jondong** [comment]
LGTM with a little nit. Thanks for the PR @chuan9

**Comment by chuan9** [comment]
@jondong Done, thanks for review

**Comment by jondong** [comment]
LGTM, @baleboy @bspencer would you please help to merge? thanks

**Pull request merged**